cfme/common/vm.py

```diff
@@ -325,6 +325,9 @@ def find_quadicon(self, from_any_provider=False, use_search=True):
         if 'Grid View' != view.toolbar.view_selector.selected:
             view.toolbar.view_selector.select('Grid View')
 
+        if use_search:
+            view.search.simple_search(self.name)
```

Comment:
This is already passed to `view.entities.get_entity` and used there, I think you can drop these two lines.